### DISCUSSION TITLE
Thank you - excellent work

### DISCUSSION THREAD

**Discussion by widert** [zwave-js/node-red-contrib-zwave-js#195] [Ideas]
Just would like to say thank you guys. A little bit of learning curve, but the final result is soooooo good!

**Comment by marcus-j-davies** [comment]
Hi @widert,
You are very welcome - and thanks. 👍 

I have 6.6.0 being worked on which will improve the UI layout and more.

again thank you!

**Reply by fabnavigator** [reply]
@widert I agree. I've migrated all of my z-wave to this, and was able to stop using Home Assistant. I feel that we are all reinventing the wheel to some extent.  I wonder if there are some basic flows that should be available to do some higher level tasks (i.e., integrating with dashboard switches, sending out notifications, processing commands from remote controls, etc.).

**Reply by marcus-j-davies** [reply]
HI @fabnavigator 

We do have examples built into the module.
But they are geared towards the nodes them self.

I purposely didn't produce any specific setup - Why?
The module is designed as a framework and this is what makes it very attractive - and it's one that allows for users to bend/shape into any number of ways.

If however, you and @amilanov75 who also uses dashboards, ever wanted to share you creations with this module and how you have wired it in - I am more than happy to include a 'User Contributed' example collection.

some key points I would like to keep in mind.
 - The examples should work 'out of the box'
 - Should include any inject node that may be needed to create global vars/environments.
 - Must state any dashboard module that may be needed.
 - Should not be complex - Should be simple to understand (using 1 device as an example)
 - If the example includes the controller node - it must be disabled (so it doesn't affect any users live environment)
 
 
The examples will be included along with the basic set.
**Import Menu -> Example -> node-red-contrib-zwave-js**

**Reply by fabnavigator** [reply]
@marcus-j-davies I like that idea. I'll work on something that I can share.

**Reply by marcus-j-davies** [reply]
@fabnavigator 

No worries.
if you look at the example I have included with the module - you will see I comment and provide info on what's what.

if we can keep it simple, uncomplicated, informative - then we can include them under 'User Contributed' examples folder, that will get packaged with the module - feel free to include contact details in comment nodes- so users using the examples, can ask you questions.
